src/community/ogcapi/ogcapi-tiles/src/main/java/org/geoserver/ogcapi/tiles/TiledCollectionDocument.java

```diff
@@ -43,7 +42,7 @@
 
 /** Description of a single collection, that will be serialized to JSON/XML/HTML */
 @JsonPropertyOrder({"id", "title", "description", "extent", "links", "styles"})
-public class TiledCollectionDocument extends AbstractCollectionDocument {
+public class TiledCollectionDocument extends AbstractCollectionDocument<TileLayer> {
```

Comment:
Does it need to be public?

Reply:
Is this comment possibly against the wrong line?

Reply:
Yep. Was wondering in general about changes of visibilty between package private and public. See comment below. No action required.